site/pages/create-disruption.tsx

```diff
@@ -10,58 +11,192 @@ interface CreateDisruptionProps {
+                                    id="description"
+                                    name="description"
+                                    rows={5}
                                 />
-                                <label className="govuk-label govuk-radios__label" htmlFor="disruption-repeats">
-                                    Yes
-                                </label>
                             </div>
-                            <div className="govuk-radios__item">
+                            <div className="govuk-form-group">
+                                <legend className="govuk-fieldset__legend govuk-!-padding-top-2">
+                                    <h3 className="govuk-heading-s govuk-!-margin-bottom-0" id="associated-link">
+                                        Associated Link
+                                    </h3>
+                                </legend>
                                 <input
-                                    className="govuk-radios__input"
-                                    id="disruption-does-not-repeat"
-                                    name="disruptionRepeats"
-                                    type="radio"
-                                    value="no"
+                                    className="govuk-input w-3/4"
+                                    id="associated-link"
+                                    name="associatedLink"
+                                    type="text"
                                 />
-                                <label className="govuk-label govuk-radios__label" htmlFor="disruption-does-not-repeat">
-                                    No
-                                </label>
+                            </div>
+                            <div className="govuk-form-group">
+                                <legend className="govuk-fieldset__legend govuk-!-padding-top-2">
+                                    <h3 className="govuk-heading-s govuk-!-margin-bottom-0" id="disruption-reason">
+                                        Reason for disruption
+                                    </h3>
+                                </legend>
+                                <select className="govuk-select w-3/4" id="disruption-reason" name="disruptionReason">
+                                    {getReasonOptions()}
+                                </select>
                             </div>
                         </div>
-                    </fieldset>
+                        <div className="govuk-form-group">
+                            <h1 className="govuk-heading-l">When is the disruption?</h1>
 
-                    <h1 className="govuk-heading-l govuk-!-padding-top-8">
-                        When does the disruption need to be published?
-                    </h1>
+                            <DisruptionsDateTimeInfo inputs={inputs} isDisruptionValidity />
+                            <legend
```

Comment:
This legend and the below radio buttons should be wrapped in a fieldset

Reply:
done